Content.Shared/DoAfter/DoAfterArgs.cs

```diff
@@ -40,17 +40,31 @@ public sealed partial class DoAfterArgs
 
     public NetEntity? NetUsed;
 
+    // Goobstation - Show doAfter progress bar to another entity
+    [NonSerialized]
+    [DataField]
```

Comment:
What is NonSerialized? Aren't these the opposite of each other?

Reply:
uHhhhhhhhh, I'm not qualified enough to answer that question and will figure it out later.